forcedotcom/source-deploy-retrieve
# Sm/archiver-bug

### What does this PR do?
don't await for archiver.  See comments for explanation

### What issues does this PR fix or reference?

@W-12050345@ https://github.com/forcedotcom/cli/issues/1802


---
QA/Repro

use https://github.com/mshanemc/nestedFolders, then switch to the sdr-768 branch.

it's already got a mdapiOut folder with an email template which contains a very, very large attachment (as suggested by the customer on https://github.com/forcedotcom/cli/issues/1802
create an org
yarn link SDR into plugin-source, then `../../plugin-source/bin/dev force:mdapi:deploy --deploydir mdapiOut --checkonly --json`


you'll see the sfdx version of the command silently exit, but the version with this SDR branch works as expected.
